jfix-stdlib-batching/src/jmh/java/ru/fix/stdlib/batching/BatchingManagerHighContentionJmh.java

```diff
@@ -0,0 +1,81 @@
+package ru.fix.stdlib.batching;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.State;
+import ru.fix.aggregating.profiler.NoopProfiler;
+
+import java.util.function.IntConsumer;
+import java.util.stream.IntStream;
+
+@State(Scope.Benchmark)
+public class BatchingManagerHighContentionJmh {
```

Comment:
Keeping so much dublicated code (classes from jmh/java BatchingManagerSemaphore, BatchingManagerThreadSemaphore, BatchProcessorSemaphore, as I understand, dublicate classes from main/java) increases code reading complexity.

Reply:
We discussed it and decided to leave comments to this jmh stating why we needed to make those dublications.

Reply:
javadoc added